core/arch/arm/include/kernel/generic_boot.h

```diff
@@ -5,6 +5,12 @@
 #ifndef KERNEL_GENERIC_BOOT_H
 #define KERNEL_GENERIC_BOOT_H
 
+#if defined(CFG_BOOT_SECONDARY_REQUEST)
+#define NSENTRY_EP_ADDR_OFFS		0
+#define NSENTRY_CONTEXT_ID_OFFS		__SIZEOF_POINTER__
```

Comment:
Please use `core/arch/arm/kernel/asm-defines.c` instead to create these defines, if they are needed at all.